.editorconfig

```diff
@@ -8,6 +8,7 @@ root = true
 charset = utf-8
 end_of_line = lf
 insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Better to split PR next time.

Reply:
Created https://github.com/Wargus/stratagus/pull/648